src/Jobs/CheckForUpdatesJob.php

```diff
@@ -48,4 +64,29 @@ public function process()
         // mark job as completed
         $this->isComplete = true;
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function afterComplete()
+    {
+        // Gather config options
+        $reschedule = Config::inst()->get(__CLASS__, 'reschedule');
+        $rescheduleDelay = Config::inst()->get(__CLASS__, 'reschedule_delay');
+
+        if ($reschedule === false) {
+            return;
+        }
+
+        // Queue a new job to run in the future
+        $injector = Injector::inst();
+        $queuedJobService = $injector->get(QueuedJobService::class);
+
+        $startAfter = new DateTime(SS_Datetime::now()->getValue());
+        $startAfter->modify('+' . $rescheduleDelay);
```

Comment:
Bug: you have `+` twice now - once in the config value and once here

Reply:
I did notice that I forgot to fix it :(